src/page/oyblikksbilde-visning/oyeblikksbilde-registrering.tsx

```diff
@@ -23,32 +27,52 @@ import {
 } from './oyblikksbilde-fikser';
 import { RegistreringDto } from './dto/RegistreringDto';
 import { FilePdfIcon } from '@navikt/aksel-icons';
-import { visEnkelVerdi } from './oyeblikksbilde-cv';
+import { visEnkelVerdi, visFlereVerdi } from './oyeblikksbilde-cv';
+import { OpplysningerOmArbeidssokerMedProfilering } from './dto/OpplysningerOmArbeidssoekerMedProfilering';
+import { lagHentTekstForSprak, SPORSMAL_TEKSTER, SporsmalId } from '@navikt/arbeidssokerregisteret-utils';
 
 export function OyeblikksbildeRegistrering(props: { vedtakId: number }): JSX.Element {
-	const oyeblikksbildeFetcher = useAxiosFetcher(hentRegistreringOyblikksbilde);
+	const registreringOyeblikksbildeFetcher = useAxiosFetcher(hentRegistreringOyblikksbilde);
+	const arbeidssokerRegistretOyeblikksbildeFetcher = useAxiosFetcher(hentArbeidssokerRegistretOyblikksbilde);
 
 	useEffect(() => {
-		oyeblikksbildeFetcher.fetch(props.vedtakId);
+		registreringOyeblikksbildeFetcher.fetch(props.vedtakId);
+		arbeidssokerRegistretOyeblikksbildeFetcher.fetch(props.vedtakId);
 		logMetrikk('vis-oyblikksbilde-registering');
 		// eslint-disable-next-line
 	}, [props.vedtakId]);
 
-	if (oyeblikksbildeFetcher.loading) {
+	if (registreringOyeblikksbildeFetcher.loading || arbeidssokerRegistretOyeblikksbildeFetcher.loading) {
 		return <Spinner />;
-	} else if (oyeblikksbildeFetcher.error) {
+	} else if (registreringOyeblikksbildeFetcher.error || arbeidssokerRegistretOyeblikksbildeFetcher.error) {
 		return (
 			<Alert variant="error" className="vedtaksstotte-alert">
 				Det oppnås for tiden ikke kontakt med alle baksystemer. Vi jobber med å løse saken. Vennligst prøv igjen
 				senere.
 			</Alert>
 		);
-	} else if (oyeblikksbildeFetcher.data) {
+	} else if (arbeidssokerRegistretOyeblikksbildeFetcher.data?.data) {
```

Comment:
er det sånn at backenden svarer med null på en av disse så du slepper å sjekke på registreringsdato her?

Reply:
Backend will respond with data = null if we dont have data but I don't understand fully what you meant